charts/aks-operator/templates/deployment.yaml

```diff
@@ -25,3 +25,17 @@ spec:
           value: {{ .Values.httpsProxy }}
         - name: NO_PROXY
           value: {{ .Values.noProxy }}
+{{- if .Values.additionalTrustedCAs }}
+        volumeMounts:
+          - mountPath: /etc/ssl/certs/ca-additional.pem
+            name: tls-ca-additional-volume
+            subPath: ca-additional.pem
+            readOnly: true
+{{- end }}
+{{- if .Values.additionalTrustedCAs }}
+      volumes:
+        - name: tls-ca-additional-volume
+          secret:
+            defaultMode: 0400
+            secretName: tls-ca-additional
```

Comment:
Do we need to specify the namespace from where the secret is?

Reply:
It looks like the api does not support setting the namespace. 
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#secretvolumesource-v1-core

I am not sure if it means we can use the secret in any namespace or only where the pod is in.

Reply:
Oh, I just realized that the aks-operator is installed into the cattle-system namespace where the `tls-ca-additional` secret is created.

Reply:
Yes, that is right. This pattern follows the same one from Rancher: https://github.com/rancher/rancher/blob/master/chart/templates/deployment.yaml#L136-L145